lib/pdftotext/cli.rb

```diff
@@ -8,7 +8,7 @@ class CLI
     def run_command(*args)
       options = DEFAULT_OPTIONS.merge(args.pop)
       args = args.concat options_to_args(options)
-      output, status = Open3.capture2e(bin_path, *args)
+      output, status = Open3.capture2e("#{bin_path} #{args.join(" ")}")
```

Comment:
`Capture2e` is intended to take an array of arguments and build a string representation of them (handling things like escaping, etc.). We shouldn't have to concatenate the string ourselves.